v0.2.3 Release

lukecampbell opened the pull request:
This commit increments the conda recipe to use version 0.2.3

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

lukecampbell committed:
v0.2.3 Release

This commit increments the conda recipe to use version 0.2.3

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

lukecampbell commented:
@conda-forge-admin, please rerender

lukecampbell committed:
Fixes SHA

PyPI apparently does not resolve the URL for the package, but has a
redirect. The script I used did not follow redirects so the HTTP 301
Moved Permanently page was used to compute the hash instead of the
actual package.
